### PR TITLE
AI service

### PR DESCRIPTION
**Pull request by RihabFekii**
I added documentation to the ML API and to the ML package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,201 @@
 # Proof-of-Concept: AI Service
 Implementation of a Proof-of-Concept of an AI service provisioned on a marketplace using i4Trust technologies
 
+## Introduction
+The aim of this PoC is to allow an AI service provider to offer a service on a public marketplace, where a service consuming party can acquire access to this offer, inject input data into it and receive predictions resulting from the inference with a trained AI model.
+
```
Comment (wistefan) [inline README.md:6]: real-time should be removed, its not real time

File: it/bin/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wistefan) [inline it/bin/.project]: local file, should be removed

File: it/bin/.settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```
Comment (wistefan) [inline it/bin/.settings/org.eclipse.core.resources.prefs]: local file, should be removed

File: it/bin/.settings/org.eclipse.jdt.core.prefs [hunk 4]
```diff
@@ -0,0 +1,576 @@
+eclipse.preferences.version=1
```
Comment (wistefan) [inline it/bin/.settings/org.eclipse.jdt.core.prefs]: local file, should be removed

File: it/bin/.settings/org.eclipse.m2e.core.prefs [hunk 5]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
Comment (wistefan) [inline it/bin/.settings/org.eclipse.m2e.core.prefs]: local file, should be removed

**Review by wistefan — COMMENTED** [review]
What are the bin and it folders for? Looks like local & compiled code, that should not be part of the repo

**Review by dwendland — APPROVED** [review]

**Comment by dwendland** [comment]
@wistefan Please review again

**Pull request merged**